android/src/test/java/io/keen/client/android/AndroidJsonHandlerTest.java

```diff
@@ -157,11 +222,51 @@ public void writeEvillyNestedMapsAndCollections() throws Exception {
         list1.add(map2);
         map1.put("o", list1);
 
+        // {
```

Comment:
I dunno, I'm open to feedback here. We don't need it but I found it very helpful in sorting out what the above code actually represents, and in building the assertions below. Happy to delete it if you think it's not helpful. I guess if we keep it I should probably add a little note explaining why it's there.